app/views/handlers/feedback.py

```diff
@@ -48,8 +48,9 @@ def __init__(
     ):
         if not self.is_enabled(schema):
             raise FeedbackNotEnabled
-
-        if self.is_limit_reached(session_store.session_data):
+        if isinstance(
+            session_store.session_data, SessionData
+        ) and self.is_limit_reached(session_store.session_data):
```

Comment:
As above.

Reply:
I've added `ignore` to it, for now, as we probably need wider discussion about it.